### DISCUSSION TITLE
Cadenas

### DISCUSSION THREAD

**Discussion by patricia2454** [LaunchX-InnovaccionVirtual/CursoIntroPython#30] [Q&A] [answered]
Saludos:
Tengo aún muchas dudas con el tema de "cadenas"; ¿dónde pudiera encontrar mayor información?

Gracias

**Comment by Ricardo-g-garcia** [comment, marked as answer]
Esa es una Referencia de los métodos de cadenas
https://www.w3schools.com/python/python_ref_string.asp

y ese va paso a paso
https://www.w3schools.com/python/python_strings.asp

**Reply by patricia2454** [reply]
Gracias

**Reply by YomairaBarrera** [reply]
Muchas gracias esta genial esa pagina
